rpc/synchronization_forcer.go

```diff
@@ -0,0 +1,42 @@
+package rpc
+
+import (
+	"errors"
+
+	"github.com/TykTechnologies/tyk/apidef"
+	"github.com/TykTechnologies/tyk/storage"
+)
+
+type SyncronizerForcer struct {
+	store *storage.RedisCluster
+}
+
+func NewSyncForcer(redisController *storage.RedisController) *SyncronizerForcer {
+	sf := &SyncronizerForcer{}
+
+	sf.store = &storage.RedisCluster{KeyPrefix: "synchronizer-group-", RedisController: redisController}
+	sf.store.Connect()
+
+	return sf
+}
```

Comment:
Could you explain why do we need a separate object and functions to force a synchronization? is not enough only sending the param to MDCB?

Reply:
@sredxny the `force` param is going to change depending if the key exists or not in the Redis. 
[The RPC `Connect`  function receives a function `getGroupLoginFunc`](https://github.com/TykTechnologies/tyk/blob/master/rpc/rpc_client.go#L205) so I thought having a separated structure with its own `groupLoginCallback` it's the cleanest way to do it, without modifying the current behavior since this new callback func is only going to work when the synchronizer is enabled (`slave_options.synchroniser_enabled`).

Does it makes sense?